cmd/snap/cmd_find.go

```diff
@@ -154,6 +155,18 @@ func (x *cmdFind) Execute(args []string) error {
 	}
 
 	cli := Client()
+
+	if x.Section != "" && x.Section != "featured" {
+		sections, err := cli.Sections()
```

Comment:
I wonder if this is an extra roundtrip to the store? And if so, if we can get the store to provide error information like "no-such-group" for us?

Reply:
Digging deeper, I think we should be able to save on the roundtrip and use the cached sections files that's already there for the completion support.